src/components/NavBar.vue

```diff
@@ -7,8 +7,16 @@
         </div>
       </router-link>
       <div class="login-container">
-        <!-- <i class="pi pi-bars menu-icon" @click="toggleMenu" /> -->
-        <!-- <PvMenu ref="menu" id="overlay_menu" :model="dropdownItems" :popup="true" /> -->
+        <div v-if="isSuperAdmin">
```

Comment:
Also, I think we can remove the surrounding div that restricts this to super admins. The `getSidebarActions` functions used below takes care of restricting the menu items that partner admins can see. But I think we still want them to see it.